src/index.js

```diff
@@ -33,42 +37,98 @@ function fetchData(searchTerm) {
         .catch(function (error) { console.log(`${error.name}:\n${error.message}`); });
 }
 
-function decodeData(foundData) {
-    for (let foundArticle of foundData.query.search) {
-        // console.log(foundArticle.snippet)
 
+async function decodeData(foundData) {
+
+    prepareResultsSection()     // create section in html where resutls will be displayed
+
+    for (let foundArticle of foundData.query.search) {
+        // console.log(foundData)
         let articleTitle = foundArticle.title
+
         let articleText = foundArticle.snippet
-        .replace(/(<([^>]+)>)/gi, "")               // strip html tags from 
-        .slice(0, 130) + "...";                      // truncate text
-        
-        let articleUrl = 'https://en.wikipedia.org/?curid=' + Number(foundArticle.pageid)        //
+            .replace(/(<([^>]+)>)/gi, "")               // strip html tags from 
```

Comment:
I really hope someone can make a good RegExp guide for me to learn tbh, such a powerful tool, but yet confusing to learn.